src/test/java/scenarios/individual/IndividualStepsCreate.java

```diff
@@ -0,0 +1,52 @@
+package scenarios.individual;
+
+import api.ApiRequestManager;
+import api.ApiResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import salesforce.endpointurl.Endpoint;
+import salesforce.entities.CreatedResponse;
+import salesforce.entities.Person;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.http.HttpStatus;
+import java.util.HashMap;
+import java.util.Map;
+
+import static salesforce.config.EnvConfigurationFile.LOGGER;
+
+public class IndividualStepsCreate {
```

Comment:
Would be good to generalize the steps so there is no need to separate the crud operations in different files.